packages/nimble-components/src/wafer-map/tests/create-matrix-renderer.spec.ts

```diff
@@ -18,19 +18,28 @@ describe('MatrixRenderer worker', () => {
     });
 
     it('updateMatrix should update the dieMatrix', async () => {
-        const testData = [4, 5, 6];
+        const testData = { colIndexes: [1, 2, 3], rowIndexes: [4, 5, 6], values: [8.1, 9.2, 10.32] };
         await matrixRenderer.updateMatrix(testData);
-        const resolvedDieMatrix = await matrixRenderer.dieMatrix;
-        expect(Array.from(resolvedDieMatrix)).toEqual(
-            Array.from(Uint8Array.from(testData))
+        const resolvedDieMatrix = await matrixRenderer.getMatrix();
+
+        expect(resolvedDieMatrix.colIndexes).toEqual(
+            Uint32Array.from(testData.colIndexes)
+        );
+        expect(resolvedDieMatrix.rowIndexes).toEqual(
+            Uint32Array.from(testData.rowIndexes)
+        );
+        expect(resolvedDieMatrix.values).toEqual(
+            Float64Array.from(testData.values)
         );
     });
 
     it('emptyMatrix should empty the dieMatrix', async () => {
-        const testData = [4, 5, 6];
+        const testData = { colIndexes: [4, 1, 2], rowIndexes: [54, 54, 62], values: [8.12, 9.0, 0.32] };
```

Comment:
If the test data contents don't really get checked against the specific values you are using (across multiple tests), then can we just declare a `testData` const ouside of the tests that we can use in multiple tests?

Reply:
indeed, I moved it outside the tests
![image](https://github.com/ni/nimble/assets/110180309/1e2561b5-6ae0-4214-8dcf-6fd21847ab30)